openshift/openshift/src/main/java/org/arquillian/cube/openshift/impl/client/CubeOpenShiftConfiguration.java

```diff
@@ -140,47 +135,4 @@ public static CubeOpenShiftConfiguration fromMap(Configuration c, Map<String, St
             }
         }
     }
-
-    public static CubeOpenShiftConfiguration fromMap(Map<String, String> map) {
```

Comment:
Yes, the CubeOpenShiftConfigurator is using a piece of code like: 

     CubeOpenShiftConfiguration cubeConfiguration = CubeOpenShiftConfiguration.fromMap(configuration, properties);